apps/predbat/config/apps.yaml

```diff
@@ -22,16 +22,20 @@ pred_bat:
   
   #
   # Sensors, currently more than one can be specified and they will be summed up automatically
-  # however if you have two inverters only set one of them as they will both read the same.
+  # If you have two inverters, each inverter will have its own load and PV generation data 
+  # but only set the import and export for one of them as these will both read the same.
```

Comment:
I'm not sure this is true, the load energy sensor should be a CT clamp and hence cover the whole house, unless you are on 3 phase and you have different inverters for each phase.

For PV it depends, if you have two PV inverters it makes sense but many people don't have PV on both.

Reply:
I've added some text about this on main now